bin/wp-merge-to-cp.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# First parameter is the changeset number
+# Second parameter is the WP branch
+
+find_wp_remote=$(git remote -v | grep WordPress/wordpress-develop | tail -n1 | awk '{print $1;}')
+if [ -z "$find_wp_remote" ]; then
+    echo "Adding WP remote"
+    # Download WordPress dev
+    git remote add wp https://github.com/WordPress/wordpress-develop > /dev/null 2>&1
+    find_wp_remote='wp'
+fi
+
+find_cp_remote=$(git remote -v | grep ClassicPress/ClassicPress | tail -n1 | awk '{print $1;}')
+if [ -z "$find_cp_remote" ]; then
+    echo "Adding CP remote"
+    # Download WordPress dev
+    git remote add cp git@github.com:ClassicPress/ClassicPress.git > /dev/null 2>&1
```

Comment:
This should use https://github.com/WordPress/wordpress-develop instead, because many people don't have git + ssh set up properly.